doc/articles/getting-started/requirements.md

```diff
@@ -1,20 +1,33 @@
-# Minimum supported target platform versions
+# Supported platforms
 
-The platform requirements for the Uno Platform are as follows:
+Uno Platform applications run on [Android](#android), [iOS](#ios), [web](#webassembly), [macOS](#macos), [Linux](#linux), Tizen, and [Windows](#windows). 
+
+See below for minimum supported versions for each platform.
 
 ## Android
 
-- The minimum supported version by the Uno Platform is Android "Lollipop" 5.0 (API 21) from 2014. Android v4.4 and lower is known to work but is unsupported.
-- Android has [several API settings](https://docs.microsoft.com/en-us/xamarin/android/app-fundamentals/android-api-levels) that determine your application's compatibility. As a rule of thumb, always [use the latest Android SDK Tools and Android API platform](https://docs.microsoft.com/en-us/xamarin/android/get-started/installation/android-sdk?tabs=windows) during development as you can target older Android versions while using the latest SDK.
+Uno Platform apps run on devices running Android 5 and above. 
+
+At compile time, Uno typically supports two versions of the Android SDK, the latest and the immediately previous. At present, this is Android 11 and Android 10. It's generally recommended to use the latest version of the SDK. (Note that this **does not** affect the runtime version - apps compiled with Android 11 will run happily on devices running Android 5.)
 
 ## iOS
 
-The minimum supported version by the Uno Platform is iOS 8 from 2014.
+Uno Platform apps run on iOS 8 and above.
```

Comment:
Is this still correct? I thought Xamarin support starts at iOS 9?

Reply:
Good catch, I did not check.

Do you have a link by any chance? Quickly googling, I didn't manage to find up-to-date information on this.

Reply:
hmm I can only find something for Xamarin.Forms https://docs.microsoft.com/en-us/xamarin/get-started/supported-platforms?tabs=macos

Reply:
checking in an Uno app, it lets me set Deployment Target in info.plist as low as 7... 

From some digging I found at least one nventive app that still has `MinimumOSVersion` set at 8.0, so I think the 9.0 minimum is X.Forms-specific. I'll leave it like this for now.